### DISCUSSION TITLE
Route Manager

### DISCUSSION THREAD

**Discussion by Antaguana** [voxeltycoon/issues#388] [UI&UX]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I want to find out information about a route, add new vehicles to one, or retire one (I just finished a research, need to stop sending iron ore to that terminal, so I need to camp outside the terminal and watch for the iron trucks to arrive so i can send them to the depot.)

**Describe the solution you'd like**
A UI that lets us:

- see all the vehicles on a route
- view stats about the route. (total delivery/profit in 30 days, ideally a graph or 2, upgrade all vehicles in route, all the usual things)
- easily clone vehicles to increase the routes throughput
- easily send all to depot.  (a partial solution for this might be to allow the user to add a depot to the route)
- find routes via a UI showing all the routes allowing various filtering
- link to/from to a terminal UI (so you can see the routes on a terminal or the terminals on a route)

**Describe alternatives you've considered**
 we can see how many vehicles are on a route by trying to add a new vehicle to it.

**Additional context**
See OTTD or Transport fever route managers...

**Comment by SMERKIN5000** [comment]
I would like:
- manage routes that have no vehicles assigned
- filter by specific station/depot (especially ones that have been destroyed)

**Comment by tbjoern** [comment]
I'd also like to be able to delete a route. The dropdown can get very crowded after a while.
